src/sentry/tasks/assemble.py

```diff
@@ -437,6 +522,27 @@ def _create_artifact_bundle(
                         values=new_date_added,
                         defaults=new_date_added,
                     )
+
+            try:
+                organization = Organization.objects.get_from_cache(id=org_id)
+            except Organization.DoesNotExist:
+                organization = None
+
+            # If we don't have a release set, we don't want to run indexing, since we need at least the release for
+            # fast indexing performance. We might though run indexing if a customer has debug ids in the manifest, since
+            # we want to have a fallback mechanism in case they have problems setting them up (e.g., SDK version does
+            # not support them, some files were not injected...).
+            if (
+                organization is not None
```

Comment:
Is there a reason this has to be `organization is not None` and not just `organization`? Could the `organization` be empty?

Reply:
I just don't like having the implicit evaluation, I prefer to explicitly check for the `not None`.

Reply:
Can't say that I disagree.